alternator/executor.cc

```diff
@@ -238,18 +238,23 @@ static std::string get_table_name(const rjson::value& request) {
  */
 schema_ptr executor::find_table(service::storage_proxy& proxy, const rjson::value& request) {
     auto table_name = find_table_name(request);
-    if (!table_name) {
```

Comment:
nit: maybe keep the the original pattern and add

`return find_table(proxy, *table_name); ` 

at the end not nested?